src/index.html

```diff
@@ -11,8 +12,74 @@
       rel="stylesheet"
       href="./styles/index.scss"
     />
+
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    />
   </head>
-  <body>
-    <h1>Product cards</h1>
+
+  <!-- #endregion -->
+
+  <body
+    class="card"
+    data-qa="card"
+  >
+    <h1 class="imac">
```

Comment:
Using a heading tag `h1` for wrapping an image is not semantically correct. If the image is a logo, it's better to use a `div` or a `header` tag.